src/web/src/app/space/detail/component.html

```diff
@@ -0,0 +1,68 @@
+<h1>{{title}}</h1>
+
+<hr class="width-and-clear">
+<div class="space-invites-heading">
+	<h2>Приглашения</h2>
+	<button mat-icon-button><mat-icon>add</mat-icon></button>
+</div>
+<div class="width-and-clear">
+	<div *ngFor="let invite of receivedInvites" class="row">
+		<div class="left-float">{{invite.user}}</div>
+		<div class="left-float left-padding">
+			<mat-form-field>
+				<select matNativeControl [(ngModel)]="invite.role">
+					<option value="admin">admin</option>
+					<option value="user">user</option>
+					<option value="guest">guest</option>
+				</select>
+			</mat-form-field>
+		</div>
+		<div class="left-float left-padding"><button mat-icon-button><mat-icon>close</mat-icon></button></div>
+	</div>
+</div>
+<div *ngIf="invites.length > 7" class="width-and-clear">
+	<mat-paginator [length]="invites.length" [pageSize]="invitesPageSize" (page)="pageEvent = getPaginatorInvites($event)">
+	</mat-paginator> 
+</div>
+
+<hr class="width-and-clear">
+<div class="space-refs-heading">
+	<h2>Ссылки-приглашения</h2>
+	<button mat-icon-button><mat-icon>add</mat-icon></button>
+</div>
+<div class="width-and-clear">
+	<div *ngFor="let ref of receivedRefs" class="row">
+		<div class="left-float">{{ref.name}}</div>
+		<div class="left-float left-padding">{{ref.date}}</div>
+		<div class="left-float left-padding"><button mat-raised-button>Копировать</button></div>
+		<div class="left-float left-padding"><button mat-icon-button><mat-icon>close</mat-icon></button></div>
+	</div>
+</div>
+<div *ngIf="refs.length > 7" class="width-and-clear">
+	<mat-paginator [length]="refs.length" [pageSize]="refsPageSize" (page)="pageEvent = getPaginatorRefs($event)">
+	</mat-paginator> 
+</div>
```

Comment:
А зачем тут ещё один div ? Пагинатор же сам уже div. 
```suggestion
<mat-paginator *ngIf="refs.length > refsPageSize"
	[length]="refs.length"
	[pageSize]="refsPageSize"
	(page)="pageEvent = getPaginatorRefs($event)">
</mat-paginator> 
```

Reply:
Поправил.